src/local/butler/deploy.py

```diff
@@ -92,7 +92,8 @@ def _deploy_app_prod(project,
                      yaml_paths,
                      package_zip_paths,
                      deploy_appengine=True,
-                     test_deployment=False):
+                     test_deployment=False,
```

Comment:
Should we delete the test_deployment feature?

Reply:
What is this feature? I have never used it

Reply:
Oh it is the test-targets thing. We could delete it if this proves successful, we leave the test cluster alive and perform our releases on it with this approach (might be a good idea to create a windows one for that purpose too)

Reply:
Now that I think of it, I would rather keep this feature. At some point in the future I would like to pin releases (git commit) in our deployments, so we can have granular control over what code is executed in each cluster.

For now, this is still useful.